hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBTables.java

```diff
@@ -336,4 +336,88 @@ public ShardIndex(String database) {
             super(database, TABLE);
         }
     }
+
+    public static class Olap extends RocksDBTable {
+
+        public static final String TABLE = HugeType.OLAP.string();
+
+        public Olap(String database, Id id) {
+            super(database, joinTableName(TABLE, id.asString()));
+        }
+
+        @Override
+        protected BackendColumnIterator queryById(Session session, Id id) {
+            return this.getById(session, id);
+        }
+
+        @Override
+        public boolean isOlap() {
+            return true;
+        }
+    }
+
+    public static class OlapSecondaryIndex extends SecondaryIndex {
+
+        public static final String TABLE = HugeType.OLAP.string();
+
+        public OlapSecondaryIndex(String store) {
+            this(store, TABLE);
+        }
+
+        protected OlapSecondaryIndex(String store, String table) {
+            super(joinTableName(store, table));
+        }
+    }
+
+    public static class OlapRangeIntIndex extends RangeIntIndex {
+
+        public static final String TABLE = HugeType.OLAP.string();
```

Comment:
is it the same table name as OlapSecondaryIndex

Reply:
no, it's like g_ap_ii